map_model/src/make/turns.rs

```diff
@@ -59,16 +59,46 @@ pub fn make_all_turns(map: &Map, i: &Intersection) -> Vec<Turn> {
         });
     }
 
+    let banned_turns = banned_turns(&all_turns, &filtered_turns);
+
     // But then see how all of that filtering affects lane connectivity.
     match verify_vehicle_connectivity(&filtered_turns, i, map) {
-        Ok(()) => filtered_turns,
+        Ok(()) => (
+            filtered_turns,
+            banned_turns
+        ),
         Err(err) => {
             warn!("Not filtering turns. {}", err);
-            all_turns
+            (all_turns, Vec::<Turn>::new())
         }
     }
 }
 
+fn banned_turns(all_turns: &Vec<Turn>, filtered_turns: &Vec<Turn>) -> Vec<Turn> {
+    // let item_set: HashSet<_> = previous_items.iter().collect();
```

Comment:
Style nit: please remove commented out code in general for merging, or maybe put in an `if false { ... }` block to keep the type checker still looking at it through future refactors